lib/logstash/outputs/zeromq.rb

```diff
@@ -107,7 +107,7 @@ def server?
 
   public
   def receive(event)
-    return unless output?(event)
+    
```

Comment:
this might be a limitation of the mass update script, but would be nicer to not add an empty line